animepahe-dl.sh

```diff
@@ -238,7 +250,53 @@ download_episode() {
     if [[ -z ${_LIST_LINK_ONLY:-} ]]; then
         print_info "Downloading Episode $1..."
         [[ -z "${_DEBUG_MODE:-}" ]] && erropt="-v error"
-        "$_FFMPEG" -headers "Referer: $_REFERER_URL" -i "$pl" -c copy $erropt -y "$_SCRIPT_PATH/${_ANIME_NAME}/${1}.mp4"
+        if [[ ${_PARALLEL_RUN:-} = "true" ]]; then
```

Comment:
I'd suggest to put them in a separate function to make the structure looks cleaner.

Reply:
where else is perfect than the download_episode function?

Reply:
anyways it is time to make this implementation the default and make the FFmpeg dowloading as a fallback, so I think it is in the right place.